dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/WebContainerRequestState.java

```diff
@@ -40,7 +37,8 @@ public class WebContainerRequestState {
     private boolean startAsync;
     private IExtendedRequest currentThreadsIExtendedRequest;
     private IExtendedResponse currentThreadsIExtendedResponse;
-    private Map<String,String> cookieAttributesMap = null;
+    // Each cookie has a Map of attributes
+    private Map<String,HashMap<String,String>> cookieAttributesMap = null;
```

Comment:
Nested map should be just be `Map`, not `HashMap`

Reply:
Created compilation errors, so I'll leave it as it is: `Map<String,HashMap<String,String>>`